package.json

```diff
@@ -2827,6 +2827,212 @@
+														"command": "csharp.listRemoteProcess"
+													}
+												]
+											},
+											{
+												"key": "g",
+												"name": "+Go to",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "d",
+														"name": "Go to definition",
+														"type": "command",
+														"command": "editor.action.revealDefinition"
+													},
+													{
+														"key": "e",
+														"name": "Go to errors/problems",
+														"type": "command",
+														"command": "workbench.action.problems.focus"
+													},
+													{
+														"key": "g",
+														"name": "Go to definition",
+														"type": "command",
+														"command": "editor.action.revealDefinition"
+													},
+													{
+														"key": "i",
+														"name": "Go to implementations",
+														"type": "command",
+														"command": "editor.action.goToImplementation"
+													},
+													{
+														"key": "r",
+														"name": "Peek references",
+														"type": "command",
+														"command": "editor.action.referenceSearch.trigger"
+													},
+													{
+														"key": "D",
+														"name": "Peek definition",
+														"type": "command",
+														"command": "editor.action.peekDefinition"
+													},
+													{
+														"key": "I",
+														"name": "Find all implementation",
+														"type": "command",
+														"command": "references-view.findImplementations"
+													},
+													{
+														"key": "R",
+														"name": "Find all references",
+														"type": "command",
+														"command": "references-view.findReferences"
+													}
+												]
+											},
+											{
+												"key": "r",
+												"name": "+Refactor",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": ".",
+														"name": "Quick fix",
+														"type": "command",
+														"command": "editor.action.quickFix"
+													},
+													{
+														"key": "r",
+														"name": "Rename symbol",
+														"type": "command",
+														"command": "editor.action.rename"
+													}
+												]
+											},
+											{
+												"key": "t",
+												"name": "+Test",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "t",
+														"name": "Run test under cursor",
+														"type": "command",
+														"command": "dotnet.test.runTestsInContext"
+													},
+													{
+														"key": "d",
+														"name": "Debug test under cursor",
+														"type": "command",
+														"command": "dotnet.test.debugTestsInContext"
+													}
+												]
+											},
+											{
+												"key": "s",
```

Comment:
aren't we using `b` as backend for this?
https://develop.spacemacs.org/layers/+tools/lsp/README.html

Reply:
Somehow c# layer uses `s`. Maybe for consistency we should move that over to `b`?
https://develop.spacemacs.org/layers/+lang/csharp/README.html#omnisharp-server-interaction